CHANGELOG.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 
 Version 2.8.0
 -------------
+- use isclose in spatial point_to_section_segment (#84)
```

Comment:
Also note the change in signature of the function

Reply:
how do I do that?

Reply:
You are a lazy one
```suggestion
- Use numpy.isclose in spatial point_to_section_segment. Add new keyword arguments ``rtol``,`` atol`` to ``point_to_section_segment``.  (#84)
```

Reply:
so long!